website_event_snippet_calendar/controllers/main.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Tecnativa - Jairo Llopis
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from datetime import date, timedelta
+from openerp.fields import Date
+from openerp.http import Controller, request, route
+
+
+class EventCalendar(Controller):
+    @route("/website_event_snippet_calendar/days_with_events",
+           auth="public", type="json")
+    def days_with_events(self, start, end):
+        """Let visitors know when are there going to be any events.
+
+        :param start string:
+            Search events from that date.
+
+        :param end string:
+            Search events until that date.
+        """
+        events = request.env["event.event"].search([
```

Comment:
What about published flag?

Reply:
It's not necessary because the permission rules already force that.

This way, authorized users can also see unpublished events. That's how upstream website events list works too.

Reply:
OK